internal/jobobject/utilities.go

```diff
@@ -0,0 +1,17 @@
+package jobobject
+
+// This file contains helpers for converting oci specific values to job object values
```

Comment:
I would really like the `jobobject` package to have no idea what a container or OCI is. It should just be an adaptation of the job object API for Go. That will allow us to easily re-use it anywhere else we want to use job objects.

Reply:
We had a similar discussion up here: https://github.com/microsoft/hcsshim/pull/931#discussion_r599677485

Curious what you think about the filename for this (but agreed the comment should go)

Reply:
I agree too. I'm not really sure where something like this would live though.